### DISCUSSION TITLE
How to run the selected web file?

### DISCUSSION THREAD

**Discussion by PioKrk** [syracine69/phpdesktop#4] [Q&A] [answered]
It will be nice to have the possibility to start the process with parameters that point to a direct .html file inside /www folder. So when the app starts it will open that page. Is it possible?

**Comment by syracine69** [comment, marked as answer]
I apology to not answer you sooner. I didn't see you post.

You can't pass a filename as parameter after the binary. But you can open PHP Desktop to a specific filename by changing the web_server and index_files property in the settings.json file located in the same directory than the binary.

If you need, you can contribute to the project by sending my a pull request.
